app/dashboards/classified_listing_category_dashboard.rb

```diff
@@ -15,6 +15,8 @@ class ClassifiedListingCategoryDashboard < Administrate::BaseDashboard
     name: Field::String,
     rules: Field::String,
     slug: Field::String,
+    social_preview_description: Field::String,
```

Comment:
As social preview fields are not in the `FORM_ATTRIBUTES`, they won't appear in the form, so how will admins edit them?

Reply:
🤦 I'll add them in a separate PR I guess, as this was merged already. Thanks @lightalloy!